docs/v2-system-addon/1.GETTING_STARTED.md

```diff
@@ -0,0 +1,74 @@
+# Activity Stream (System add-on)
+
+## Where to ask questions
+
+- Most of the core dev team can be found on the `#activity-stream` mozilla irc channel.
+  You can also direct message the core team (`dmose`, `emtwo`, `jkerim`, `k8hudson`, `Mardak`, `nanj`, `r1cky`, `ursula`)
+  or our manager (`tspurway`)
+- Our Mozilla slack channel is #activitystream
+- File issues/questions on Github: https://github.com/mozilla/activity-stream/issues
```

Comment:
A sub note that we typically triage every Monday to give some expectation of turnaround.